// Copyright (C) 2023 db3.network Author imotai <codego.me@gmail.com>
// SPDX-FileCopyrightText: 2023 imotai <jackwang@octogen.dev>
// SPDX-FileContributor: imotai
//
// SPDX-License-Identifier: Elastic-2.0

syntax = "proto3";

package octopus_common_proto;

message FileChunk {
  bytes buffer = 1;
  string filename = 2;
}

message FileUploaded {
  int32 length = 1;
}

message DownloadRequest {
  string filename = 1;
}
